flink-table/flink-sql-parser/pom.xml

```diff
@@ -68,12 +68,12 @@ under the License.
 			<version>${calcite.version}</version>
 			<exclusions>
 				<!--
-				"mvn dependency:tree" as of Calcite 1.34.0:
+				"mvn dependency:tree" as of Calcite 1.35.0:
 
-				[INFO] +- org.apache.calcite:calcite-core:jar:1.34.0:compile
+				[INFO] +- org.apache.calcite:calcite-core:jar:1.35.0:compile
 				[INFO] |  +- org.apache.calcite.avatica:avatica-core:jar:1.23.0:compile
 				[INFO] |  +- org.apiguardian:apiguardian-api:jar:1.1.2:compile
-				[INFO] |  +- org.checkerframework:checker-qual:jar:3.12.0:compile
+				[INFO] |  +- org.checkerframework:checker-qual:jar:3.10.0:compile
```

Comment:
this is not related to this commit, however seems at some point the version was downgraded in Flink.
There is no critical changes, so should be ok